### DISCUSSION TITLE
What is the use of tests scripts.

### DISCUSSION THREAD

**Discussion by Atharv181** [EddieHubCommunity/LinkFree#2179] [Q&A] [answered]
What is the use of test scripts and what does fixme function do I only see comments in test.fixme() function.

**Comment by eddiejaoude** [comment, marked as answer]
The tests drive the browser like user, you can learn more on the official website http://playwright.dev

`fixme()` tests are not complete, they are tests that need to be implemented still

**Reply by Atharv181** [reply]
Thank you eddie!
Will go through the site
